modules/ROOT/pages/architecture.adoc

```diff
@@ -44,14 +44,20 @@ Mule applications run on workers. Multiple replicas of applications can run acro
 
 == Scaling Considerations
 
-* Required number of controller nodes. The maximum number of controller nodes supported is 3.
+* Minimum required number of controller nodes. 
```

Comment:
Terminating punctuation only for complete sentences.

Reply:
Got it - thank you!